server.js

```diff
@@ -30,6 +32,26 @@ const makeSgRequest = body =>
     body: body.toJSON()
   })
 
+// This converts {a:1, b:2} into 'a=1&b=2'
+const queryParams = obj =>
+  Object.keys(obj)
+    .map(key => [key, obj[key]]) // There is no Object.entries() in node 6
+    .map(([key, val]) => `${key}=${val}`)
+    .join('&')
+
+const wrikeMkFolder = name =>
+  fetch(process.env.WRIKE_URL, {
+    body: queryParams({
```

Comment:
Might want to wrap this in a call to `encodeURIComponent`. This data is coming in through a `POST` request and Wrike is expecting it to be encoded. At any rate, if you get any more weird problems with Wrike, that could be a source of errors without it.